js/signin.js

```diff
@@ -1,118 +1,72 @@
-const querySelector = (selector) => document.querySelector(selector);
+import { emailInput, passwordInput, emailErrorMessage, passwordErrorMessage, loginForm, eyeOffButton, eyeOnButton } from "./elements.js";
+import { isValidEmail, showErrorMessage, hideErrorMessage, togglePasswordVisibility } from "./sign.js";
 
-const emailInput = querySelector("#email-input");
-const passwordInput = querySelector("#password-input");
-const emailErrorMessage = querySelector("#email-error-message");
-const passwordErrorMessage = querySelector("#password-error-message");
-const loginForm = querySelector("#login-form");
-
-const eyeOffButton = querySelector("#eye-off");
-const eyeOnButton = querySelector("#eye-on");
-
-const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
+document.addEventListener("DOMContentLoaded", () => {
+  const accessToken = localStorage.getItem("accessToken");
+  if (accessToken) {
+    window.location.href = "/folder.html";
+  }
+});
 
 // 이메일 입력
-emailInput.addEventListener("focusout", validateEmailInput);
-
-function validateEmailInput() {
+emailInput.addEventListener("focusout", () => {
   const email = emailInput.value.trim();
```

Comment:
emailInput에 focusout 이벤트가 등록되어 있기 때문에 이벤트 객체로 emailInput값을 가져와도 괜찮을 것 같아요!
`DOM에 직접 접근` vs `이벤트 객체로 접근`의 고민일 것 같은데요!
어떤 차이가 있을지 (결과는 같으나 코드를 읽는 관점에서) 고민해봐도 좋을 것 같아요!
```suggestion
  const email = e.target.value.trim();
```